### ISSUE TITLE
90% error rate after 6%

### ISSUE THREAD

**Issue by vaasily** [Microsoft/CNTK#1648]
Hi,

I run the model to classify the sequence of images (CNN + LSTM). The model successfully trains for a large number of epochs, but then the error rate drops sharply to 80-90%. The subsequent training does not give any improvement. I could not understand what was going on. Can someone suggest how to fix it or what to pay attention to?

Here's how the training looks:

 Minibatch[ 501- 510]: loss = 0.147863 * 268802, metric = 5.73% * 268802;
 Minibatch[ 511- 520]: loss = 0.152009 * 269010, metric = 5.86% * 269010;
 Minibatch[ 521- 530]: loss = 0.160651 * 269108, metric = 6.07% * 269108;
 Minibatch[ 531- 540]: loss = 0.158670 * 268956, metric = 6.04% * 268956;
 Minibatch[ 541- 550]: loss = 0.169813 * 268657, metric = 6.56% * 268657;
 Minibatch[ 551- 560]: loss = 0.146374 * 269131, metric = 5.68% * 269131;
 Minibatch[ 561- 570]: loss = 0.158784 * 269042, metric = 6.19% * 269042;
 Minibatch[ 571- 580]: loss = 0.959611 * 268218, metric = 22.15% * 268218;
 Minibatch[ 581- 590]: loss = 3.999955 * 269326, metric = 82.65% * 269326;
 Minibatch[ 591- 600]: loss = 3.968455 * 269013, metric = 82.51% * 269013;
 Minibatch[ 601- 610]: loss = 3.957745 * 269154, metric = 82.62% * 269154;
 Minibatch[ 611- 620]: loss = 3.904076 * 269040, metric = 82.47% * 269040;
 Minibatch[ 621- 630]: loss = 3.881147 * 269305, metric = 82.37% * 269305;
 Minibatch[ 631- 640]: loss = 3.861322 * 269210, metric = 82.48% * 269210;

Thanks.

**Comment by nSeck** [comment]
Hey vaasily, sadly I have no idea about your problem. But I'm trying to implement a CNN+LSTM for image sequences myself. Implementation does not work yet (#1611). Do you have any advice for me how to construct input- and label-variable with dynamic axes for batch and sequence dimension in python? I would really appreciate your help!

**Comment by Ark-kun** [comment]
The model might have climbed up a hill due to momentum or a step that was too big.
Try using momuntum=0.
Try using lower Learning Rate.

**Comment by n17s** [comment]
@Ark-kun's advice is right. In addition, you can specify gradient clipping in the learner which can help prevent these issues too (especially with long recurrences).

I'm closing this as it has been quite a while. Feel free to reopen if necessary.
